examples/svdiff/pipeline_stable_diffusion_ddim_inversion.py

```diff
@@ -0,0 +1,250 @@
+from typing import Any, Callable, Dict, List, Optional, Union
+import PIL
+import torch
+from diffusers import StableDiffusionPipeline, DDIMInverseScheduler
+from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion_img2img import preprocess
+from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion_pix2pix_zero import Pix2PixInversionPipelineOutput
+
+
+class StableDiffusionPipelineWithDDIMInversion(StableDiffusionPipeline):
```

Comment:
I think we can make use of the [`DDIMInverse` scheduler](https://huggingface.co/docs/diffusers/api/schedulers/ddim_inverse) with the `StableDiffusionPipeline` here, no? 

Here is an example: https://huggingface.co/docs/diffusers/api/pipelines/stable_diffusion/pix2pix_zero#based-on-an-input-image

Reply:
Yeah, that makes sense!